test/input.txt

```diff
@@ -96,7 +96,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case-insensitive
```

Comment:
While it is good that you updated the test case, do you notice something about this block and the next block of tests? Hint: one block checks for invalid matches, and the other checks for valid matches...

Reply:
Thanks for pointing out. This is fixed in the next commit.